### PR TITLE
Cancel grandchild tweens

### PR DESCRIPTION
**Pull request by Geokureli**
```hx
FlxTween.tween(camera, {'scroll.y': 500}, 1.0);
```
The tween above can already be cancelled, via:
```hx
FlxTween.cancelTweensOf(camera.scroll);
```
but because the camera is the actual tween target, it should also be cancelled with:
```hx
FlxTween.cancelTweensOf(camera);
```

Whereas `tween(camera.scroll, {y: 500}, 1.0)` is not a tween on camera, so it must be cancelled with `cancelTweensOf(camera.scroll)` or `cancelTweensOf(camera, ['scroll.y'])`.

### CODE REVIEW

File: tests/unit/src/flixel/FlxCameraTest.hx [hunk 1]
```diff
@@ -39,16 +39,19 @@ class FlxCameraTest extends FlxTest
 	@Test
 	function testDefaultCameras():Void
 	{
-		Assert.areEqual(FlxG.cameras.defaults, FlxCamera.defaultCameras);
+		@:privateAccess
```
Comment (Gama11) [inline tests/unit/src/flixel/FlxCameraTest.hx:42]: Why start using privateAccess in here?

Comment (Geokureli) [reply tests/unit/src/flixel/FlxCameraTest.hx:42]: to remove the deprecation warnings

Comment (Gama11) [reply tests/unit/src/flixel/FlxCameraTest.hx:42]: That feels very hacky. I think I'd rather have the deprecation warnings.

Comment (Geokureli) [reply tests/unit/src/flixel/FlxCameraTest.hx]: both seem hacky, but sure

**Pull request merged**